server.js

```diff
@@ -3452,6 +3452,53 @@ cache(function(data, match, sendBadge, request) {
     }
   });
 }));
+// Magnum CI integration
+camp.route(/^\/magnumci\/([^\/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var projectId = match[1];
+  var format = match[3];
+  var branch = match[2];
```

Comment:
Could you define the parameters in their match order?
Additionally, could you put in comments an example of what they might be matched to?